docs/related/amf-conversion.mdx

```diff
@@ -1,9 +1,193 @@
+    - for example, converting to OAS to use swagger-ui in a microservice or using a trusted client-spec generator that only works with RAML.
+
+
+
+# Conversion limitations
+The AMF team, having listened to other team’s needs, is the one that dictates how the model is adapted to fit other specs:
+- If there is 1:1 information type mapping between specs it will be converted.
+- If there are missing mandatory fields default value will be added. The default’s value is opinionated.
+- Anything else is omitted.
+
+
+### Concepts present in a spec but nonexistent in another aren’t supported
+Sometimes a spec may introduce concepts that don’t exist in other specs. There are many cases coming from Raml to Oas, some are:
+- Datatype Fragments
+- Libraries
+- Resource Types and Traits
+- Overlays and Extensions
+
+These document types and objects cannot be converted to Oas as these constructs do not exist in that spec.
+These constructs can be migrated to Oas if and only if they are referenced from a root Api document.
+
+
+### Not all specs are supported
+
+This is a direct consequence of the previous limitation, not all specs can be converted between each other. Currently, Async API is not supported for conversion.
+It’s event-led concepts don’t have a direct correspondence to REST concepts described in RAML or OAS.
+For example, how would an Async APIs “subscribe” or “binding” be in RAML? The amount of similar concepts is simply not enough to create similar graphs.
+Supported conversions are:
+- RAML 1.0 to OAS 2.0 and vice-versa
+- RAML 1.0 to OAS 3.0 and vice-versa
+
+
+### Information may be lost
+Original information that is incompatible with the target spec may be lost. This is also a consequence of the first limitation.
+For example OAS 3 links have no way of being migrated into RAML and will not be rendered in the converted RAML spec.
+
+In the next example, not all of the RAML  `documentation` nodes can be migrated to OAS 3.0 `externalDocs` nodes.
+In fact, only the first `documentation` node is kept while the others are lost.
+As the RAML 1.0 `documentation` does not define an url like OAS 3.0’s does, an url node is defined with an “empty” url.
+
+
+
+```yaml title="RAML to OAS 3.0 documentation conversion example - RAML code"
+documentation:
+    - title: Test Console and Mocking Service
+    content: |
+        Welcome to the \_Test API\_ Documentation. The \_Test API\_
+        allows you to test console and mocking service features
+        [integration libraries](https://mulesoft.com)
+    - title: Legal
+    content: !include docs/api.md
+```
+
+```yaml title="RAML to OAS 3.0 documentation conversion example - OAS code"
+externalDocs:
+    url: http://
+    description: |
+        Welcome to the \_Test API\_ Documentation. The \_Test API\_
+        allows you to test console and mocking service features
+        [integration libraries](https://mulesoft.com)
+```
+
+### Conversion is not customizable
```

Comment:
Am I understanding correctly that a different conversion resolution can be created that **is** customizable? If so it would be clearer and more correct for this title to read:
"Default conversion is not customizable"

Reply:
Asked to comment on this by @arielmirra. 

Yes, I agree with the change. The conversion that is nowadays provided to the client is not customizable, we can call this the "default" one. To perform a custom conversion, the client has to create a different _resolution pipeline_ that is different from the one we provide.

Reply:
Thank you Chris and @tomsfernandez for your input, I've rewritten this part to make it clearer and also referenced to Resolution AMF documentation.